system/sefcontext.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+
+# (c) 2016, Dag Wieers <dag@wieers.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: sefcontext
+short_description: Manages SELinux file context mapping definitions
+description:
+     - Manages SELinux file context mapping definitions
+version_added: "2.2"
+options:
+  target:
+    description:
+      - Target path (expression).
+    required: true
+    default: null
+    aliases: ['path']
+  ftype:
+    description:
+      - File type.
+    required: false
+    default: a
+  setype:
+    description:
+      - SELinux type for the specified target.
+    required: true
+    default: null
+  seuser:
+    description:
+      - SELinux user for the specified target.
+    required: false
+    default: null
+  selevel:
+    description:
+      - SELinux range for the specified target.
+    required: false
+    default: null
+    aliases: ['serange']
+  state:
+    description:
+      - Desired boolean value.
+    required: false
+    default: present
+    choices: [ 'present', 'absent' ]
+  reload:
+    description:
+      - Reload SELinux policy after commit.
+    required: false
+    default: yes
+notes:
+   - The changes are persistent across reboots
+requirements: [ 'libselinux-python', 'policycoreutils-python' ]
+author: Dag Wieers
+'''
+
+EXAMPLES = '''
```

Comment:
If there is a cli tool invocation thats similar, it may be helpful to reference them in the docs, just to provide some context.

(also, not a required revision)